utils/keys_test.go

```diff
@@ -298,20 +373,80 @@ func TestImportKeys2InOneFile(t *testing.T) {
 
 	in := bytes.NewBuffer(byt)
 
-	err := ImportKeys(in, []Importer{s})
+	err := ImportKeys(in, []Importer{s}, "", "", passphraseRetriever)
+	require.NoError(t, err)
+
+	bFinal, bRest := pem.Decode(s.data["ankh"])
+	require.Equal(t, b.Bytes, bFinal.Bytes)
+	_, ok := bFinal.Headers["path"]
+	require.False(t, ok, "expected no path header, should have been removed at import")
+
+	b2Final, b2Rest := pem.Decode(bRest)
+	require.Equal(t, b2.Bytes, b2Final.Bytes)
+	_, ok = b2Final.Headers["path"]
+	require.False(t, ok, "expected no path header, should have been removed at import")
+	require.Len(t, b2Rest, 0)
+
+	cFinal, cRest := pem.Decode(s.data["morpork"])
+	require.Equal(t, c.Bytes, cFinal.Bytes)
+	_, ok = cFinal.Headers["path"]
+	require.False(t, ok, "expected no path header, should have been removed at import")
+	require.Len(t, cRest, 0)
+}
+
+func TestImportKeys2InOneFileNoPath(t *testing.T) {
+	s := NewTestImportStore()
+
+	b := &pem.Block{
+		Headers: make(map[string]string),
+	}
+	from, _ := os.OpenFile("../fixtures/secure.example.com.key", os.O_RDONLY, notary.PrivKeyPerms)
+	b.Bytes, _ = ioutil.ReadAll(from)
+	rand.Read(b.Bytes)
+	b.Headers["path"] = "ankh"
```

Comment:
Should one of these be missing a path to test what happens if one of the keys does not have a path specified?  Otherwise this test seems functionally the same as `TestImportKeys2InOneFile`  (importing multiple keys in one file that all have paths specified)